src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewIconCell.swift

```diff
@@ -33,10 +34,14 @@ class DynamicTableViewIconCell: UITableViewCell {
 		contentTextLabel.adjustsFontForContentSizeCategory = true
 		contentTextLabel.textColor = .enaColor(for: .textPrimary1)
 		contentTextLabel.numberOfLines = 0
+		contentTextLabel.lineBreakMode = .byWordWrapping
 		contentTextLabel.setContentCompressionResistancePriority(.required, for: .vertical)
 		stackView.addArrangedSubview(contentTextLabel)
 
+		// We need to make the contentTextLabel bigger when we use a bigger font e.g. title1 or title2
+		let size = (contentTextLabel.style?.fontSize ?? width) * 4
 		NSLayoutConstraint.activate([
+			contentTextLabel.heightAnchor.constraint(greaterThanOrEqualToConstant: size),
```

Comment:
Will this affect other cells besides the settings screen?

Reply:
No, the others screen using the IconCell never pass a nil text and body.

Reply:
I reverted this and restored the old state. I retested it and it worked without my change. I think it was something from the beginning i forgot to remove it.
Good eye!